packages/odyssey-storybook/src/components/odyssey-mui/DataTable/DataTable.mdx

```diff
@@ -418,6 +422,64 @@ Action buttons and menu items can be added to each row. If rows are selectable,
 which will be applied to all selected rows. (For example, the bulk actions menu starts out disabled, but if the user selects 3 rows,
 the actions menu will be enabled and any actions performed would be applied to all 3 selected rows.)
 
+## Custom filters
+
+<Canvas>
+  <Story of={DataTableStories.CustomFilters} />
+</Canvas>
+
+Within `<DataTable>`, you can now pass a custom `filters` prop. _This will override the default filters, which are based off the table columns_.
+
+The `filters` variable should be an array, which can contain any combination of:
+
+1. A table column (the full column def)
+2. A string that matches the `accessorKey` of a table column
+3. A custom filter.
+
+Using this schema, you can pass in the existing set of table columns plus your own filters. You can also use this mechanism to change the order that the filters are displayed.
+
+```tsx
+const filters = [
+  ...columns,
+  {
+    id: "",
+    label: "",
+    render: (updateFilters: UpdateFiltersCallback) => {},
+  },
+];
```

Comment:
Ah, I see, you made it super easy to add columns as filters. Uses the same API! 👍

But we're adding this `render` function that doesn't exist on the columns.